### PR TITLE
Adding CSS to go with #8

### PR DESCRIPTION
**Pull request by thatweirdcoder**
Added the css for the footer I added in the HTML

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -45,3 +45,40 @@ a {
...
 }
+/******************************
+footer
+******************************/
+footer
+{
+        padding-bottom:10px;
+        color: white;
+}
+footer p,h3,h4,h2
```
Comment (kennethlove) [inline main.css:56]: This will affect *all* `h2`, `h3`, and `h4` tags instead of just the ones in a `footer` tag.

File: main.css [hunk 2]
```diff
@@ -45,3 +45,40 @@ a {
...
+}
+.imgfoot
+{
+        clear: both;
+        margin:0 auto;
+        width:100%;
+        text-align:center;
+        padding:0 auto;
+}
+.footerContactMe
```
Comment (kennethlove) [inline main.css:80]: Three classes, three different naming conventions

File: main.css (hunk 1 above)
Comment (thatweirdcoder) [reply main.css:56]: Thanks for correcting me. :D

**Comment by lethargilistic** [comment]
I'm taking another look at these. FYI for the future, you don't have to open a second PR to update a PR (#8). You can commit directly to your PR's branch and it will update us automatically.

As it is, I'm going to try to incorporate these changes into #8 and then merge that, so this one will probably just end up closed.

**Comment by lethargilistic** [comment]
Merged directly. See #8

**Pull request closed without merging**